lib/common/client/middleware/request/rescue_timeout.rb

```diff
@@ -7,16 +7,17 @@ module Request
         class RescueTimeout < Faraday::Middleware
           include SentryLogging
 
-          def initialize(app = nil, error_tags_context = {})
+          def initialize(app = nil, error_tags_context = {}, timeout_key = nil)
             @error_tags_context = error_tags_context
+            @timeout_key = timeout_key
             super(app)
           end
 
           def call(env)
             @app.call(env)
-          rescue Faraday::TimeoutError, Net::ReadTimeout,
-                 HTTPClient::ReceiveTimeoutError, Net::OpenTimeout,
-                 EVSS::ErrorMiddleware::EVSSBackendServiceError => e
+          rescue Faraday::TimeoutError, HTTPClient::ReceiveTimeoutError,
+                 EVSS::ErrorMiddleware::EVSSBackendServiceError, Timeout::Error => e
```

Comment:
Replaced `Net::ReadTimeout` and `Net::OpenTimeout` with `Timeout::Error` since they are descendants of `Timeout::Error`